src/syntax/parser.rs

```diff
@@ -376,7 +377,7 @@ impl<TokType: Tokenizer + Iterator<Item=Tokens>> Parser<TokType> {
 
         let expr = Expr::FnDecl(fn_name, args, return_type, definition);
 
-        return Some(ExprWrapper::default(expr));
+        Some(ExprWrapper::default(expr))
```

Comment:
bleh, i personally only prefer not including the return, when you're dealing with single statements per function. like `if cond { Some() } else { None }` or `{ match cond {...None}}` or whatever. not including it for functions with branching logic and stuff makes it more confusing to me